plugins/kn-admin/main.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import "github.com/knative/client-contrib/plugins/kn-admin/core"
+
+func main() {
+  core.NewAdminCommand().Execute()
```

Comment:
What if `Execute()` fails? You need to handle error?

Reply:
Good catch! Fixed!